### DISCUSSION TITLE
competing input devides (locally and remote) on rustdesk sessions

### DISCUSSION THREAD

**Discussion by subscribernamegoeshere** [rustdesk/rustdesk#6622] [Feature Request]
I come from anydesk world, they solved the competition and "mess" with multiple people controlling one mouse cursor and one text/keyboard entry in a way that the local real physical session (or more abstract way, the non-anydesk session) has  much higher priority and overrides (timeout, time delay for the non-local input methods) over the other ones. So that the owner of a machine or the one person can not be kind of overwhealmed by remote inputs.

this would be a great addition in some way to rustdesk as well. maybe configurable about a time delay, or maybe in connection with reverting the remote side back to point-and-show style only and so forth.

thanks much for consideration.

**Comment by rxsto** [comment]
Just want to emphasize how important this is for cooperative work, e.g. coding on a project together. It's a huge mess if the remote controller has full power over the input devices without any way of stepping in for the local user. I myself and lots of my colleagues really need this feature, because it is a total game changer and forces some of them back to AnyDesk.
Cheers

**Comment by Kurgan-** [comment]
While I understand your needs, I have the exact opposite problem: when I remotely assist a user they tend to wiggle the mouse randomly because... because they do.  This makes working with Anydesk a pain because I constantly lose control over the mouse cursor. Telling them to leave the mouse alone seldom works, they keep moving it.

In Rustdesk it's less of a mess because I can "assert dominance" and wrestle back control immediately.

**Reply by open-trade** [reply]
Nothing can make every one happy.

**Reply by 857074609** [reply]
> While I understand your needs, I have the exact opposite problem: when I remotely assist a user they tend to wiggle the mouse randomly because... because they do. This makes working with Anydesk a pain because I constantly lose control over the mouse cursor. Telling them to leave the mouse alone seldom works, they keep moving it.
> 
> In Rustdesk it's less of a mess because I can "assert dominance" and wrestle back control immediately.

If it were me, I’d angrily disconnect the remote session and wait for them to apologize. It’s truly exhausting dealing with foolish people

**Reply by PC-Support-Services** [reply]
You do all realise that in RD & AD there is a tick box to block remote input?
